File: Bars.GkhGJI/libs/B4/model/BaseInsCheck.js
Ext.define('B4.model.BaseInsCheck', {
    extend: 'B4.model.InspectionGji',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'BaseInsCheck'
    },
    fields: [
        { name: 'Municipality', defaultValue: null },
        { name: 'Plan', defaultValue: null },
        { name: 'RealityObjectId', defaultValue: null },
        { name: 'InspectorId', defaultValue: null },
        { name: 'InspectionNumber' },
        { name: 'InsCheckDate' },
        { name: 'DateStart' },
        { name: 'Area' },
        { name: 'Reason' },
        { name: 'ContragentName' },
        { name: 'TypeBase', defaultValue: 10 },
        { name: 'TypeFact', defaultValue: 10 },
        { name: 'InspectionDate' },
        { name: 'DisposalNumber' },
        { name: 'InspectorNames' },
        { name: 'Address' },
        { name: 'TypeDocument', defaultValue: 10 },
        { name: 'DocumentDate' },
        { name: 'DocFile' },
        { name: 'DocumentNumber' },
        { name: 'State', defaultValue: null },
        { name: 'MoSettlement' },
        { name: 'PlaceName' }
    ]
});

File: Bars.GkhGJI/Entities/Inspection/BaseInsCheck.cs
namespace Bars.GkhGji.Entities
{
    using System;

    using Bars.B4.Modules.FileStorage;
    using Bars.GkhGji.Enums;

    /// <summary>
    /// Основание инспекционная проверка ГЖИ
    /// </summary>
    public class BaseInsCheck : InspectionGji
    {
        /// <summary>
        /// План инспекционных проверок
        /// </summary>
        public virtual PlanInsCheckGji Plan { get; set; }

        /// <summary>
        /// Дата 
        /// </summary>
        public virtual DateTime? InsCheckDate { get; set; }

        /// <summary>
        /// Дата начала
        /// </summary>
        public virtual DateTime? DateStart { get; set; }

        /// <summary>
        /// Площадь МКД (кв. м)
        /// </summary>
        public virtual Decimal? Area { get; set; }

        /// <summary>
        /// Срок проверки (Количество дней)
        /// </summary>
        public virtual int? CountDays { get; set; }

        /// <summary>
        /// Причина
        /// </summary>
        public virtual string Reason { get; set; }

        /// <summary>
        /// Факт проверки
        /// </summary>
        public virtual TypeFactInspection TypeFact { get; set; }

        /// <summary>
        /// Тип документа
        /// </summary>
        public virtual TypeDocumentInsCheck TypeDocument { get; set; }

        /// <summary>
        /// Дата документа
        /// </summary>
        public virtual DateTime? DocumentDate { get; set; }

        /// <summary>
        /// Файл
        /// </summary>
        public virtual FileInfo DocFile { get; set; }

        /// <summary>
        /// Номер документа
        /// </summary>
        public virtual string DocumentNumber { get; set; }

        /// <summary>
        /// Не хранимое поле Идентификатор Дома.
        /// </summary>
        public virtual long? RealityObjectId { get; set; }

        /// <summary>
        /// Не хранимое поле Идентификатор Инспектора.
        /// </summary>
        public virtual long? InspectorId { get; set; }
    }
}